amd_openvx_extensions/amd_rpp/source/tensor/SequenceRearrange.cpp

```diff
@@ -187,8 +187,10 @@ static vx_status VX_CALLBACK initializeSequenceRearrange(vx_node node, const vx_
 static vx_status VX_CALLBACK uninitializeSequenceRearrange(vx_node node, const vx_reference *parameters, vx_uint32 num) {
     SequenceRearrangeLocalData *data;
     STATUS_ERROR_CHECK(vxQueryNode(node, VX_NODE_LOCAL_DATA_PTR, &data, sizeof(data)));
+    if(data->pNewOrder) free(data->pNewOrder);
+    delete (data->pSrcDesc);
+    delete (data->pDstDesc);
```

Comment:
Please add space after if, free if applicable
if we should not have space, remove the space after delete in L191 - L192